project-resources/PROJECT_FUNDS.md

```diff
@@ -0,0 +1,24 @@
+# OpenJS Foundation Project-specific Initiative Funds
+
+The OpenJS Foundation provides a set of monetary funds for projects of the OpenJS Foundation. 
+These funds are designed to support collaboration and incentive project-specific initiatives that have high-impact within the individual project's scope.
+
+Alongside these lines, the fund exist to support the development of these 
+initiatives and give visibility/recognition to the champions of these project-specific initiatives, highligthing the individuals behind them.
```

Comment:
> and give visibility/recognition to the champions of these project-specific initiatives, highligthing the individuals behind them.

I don't think that should be the purpose of this fund. I would suggest simplifying that section (or anchoring back to principles that aren't subject to a controversy.).

Reply:
Hmm, that's what I've understood from what @mcollina wrote on the original issue. Maybe bad phrasing from my side?

Reply:
I don't think I ever said that.
In fact, it's the other way - funds will be needed where our volunteer-based model fails.

Reply:
@mcollina it is from this paragraph that I took what I wrote " Volunteers are driven by their motives but usually require visibility/recognition of their work. However, this model fails for tasks without recognition: if you do your job well, no one would know you did it."

I might have misunderstood or just poorly phrased my statements. Could you suggest an alternative patch here for the PR? Thank you!

Reply:
Regardless, I don't think there's a benefit in attaching a value statement of that nature to the disbursement of funds.

Reply:
I'm fine with any path, just tried to give an introductory text about what the project funds are and why they exist. Sounds like it was not a good text, if anybody has a better suggestion please comment, I'd love to have this aligned with what we believe that fits better here 🙃

Reply:
Didn't mean to be negative about your input. Apologies if it came across that way. I'm very thankful that you got this started.

Reply:
Oh no, it's fine, I just honestly don't know what we should replace the current entry with 👀

Reply:
I believe Matteo already applied a patch here 🎉